src/components/ActionBar/src/InlineActionBar.vue

```diff
@@ -47,6 +69,16 @@ export default {
 	background-image: linear-gradient(rgba(255, 255, 255, 0), rgba(255, 255, 255, 1));
 }
 
+.ActionBarWrapper {
```

Comment:
Eventually we'll need to account for this margin spacing and how it's calculated in case there's some scenarios in which it might change, but more of a mental note -- this is fine for now.